Cargo.toml

```diff
@@ -191,6 +191,7 @@ lto = true
 [profile.mobile-release]
 inherits = "mobile"
 codegen-units = 1
+lto = false
```

Comment:
any reason why you guys wanted to do this? I thought when we turned this on it made the binary considerably smaller